initiator/setup-wazo.py

```diff
@@ -164,6 +165,34 @@ def main():
     }
     incall_context = confd_client.contexts.create(body)
 
+    body = {
+        'label': 'loadtester',
+        'name': 'loadtester',
+        "templates": [{"uuid": global_sip_template_uuid}],
+        'auth_section_options': [
+            ["username", "loadtester"],
+            ["password", "loadtester"],
+        ],
+        "outbound_auth_section_options": [
+            ["username", "loadtester"],
+            ["password", "loadtester"],
+        ],
+    }
+    trunk_endpoint_sip = confd_client.endpoints_sip.create(
+        body, tenant_uuid=tenant['uuid']
+    )
+
+    body = {
+        'name': 'loadtester',
+        'context': incall_context['name'],
+        'outgoing_caller_id_format': '+E164',
+    }
+    load_tester_tunk = confd_client.trunks.create(body, tenant_uuid=tenant['uuid'])
+
+    confd_client.trunks.relations(load_tester_tunk['id']).add_endpoint_sip(
```

Comment:
confd-client doesn't need to use `.relations` nor explicit `id` (it will be extracted automatically)
```suggestion
    confd_client.trunks(load_tester_tunk).add_endpoint_sip(
```